Packs/Redmine/Integrations/Redmine/Redmine.py

```diff
@@ -443,146 +500,137 @@ def delete_issue_by_id_command(client: Client, args: dict[str, Any]):
+                                                                     is_auto_json_transform=True),
+                                     )
+    return command_results
 
 
 def get_custom_fields_command(client: Client, args):
-    try:
-        response = client.get_custom_fields_request()
-        custom_fields_response = response['custom_fields']
-        headers = ['id', 'name', 'customized_type', 'field_format', 'regexp', 'max_length', 'is_required', 'is_filter',
-                   'searchable', 'trackers', 'issue_categories', 'enabled_modules', 'time_entry_activities',
-                   'issue_custom_fields']
-        '''Some custom fields are numbers and tableToMarkdown can't transform it'''
-        for custom_field in custom_fields_response:
-            custom_field['id'] = str(custom_field['id'])
-            custom_field['is_required'] = str(custom_field['is_required'])
-            custom_field['is_filter'] = str(custom_field['is_filter'])
-        command_results = CommandResults(outputs_prefix='Redmine.CustomField',
-                                         outputs_key_field='id',
-                                         outputs=custom_fields_response,
-                                         raw_response=custom_fields_response,
-                                         readable_output=tableToMarkdown('Custom Fields List:', custom_fields_response,
-                                                                         headers=headers,
-                                                                         removeNull=True,
-                                                                         headerTransform=underscoreToCamelCase,
-                                                                         is_auto_json_transform=True
-                                                                         )
-                                         )
-        return command_results
-    except Exception as e:
-        if 'Error in API call [422]' in e.args[0] or 'Error in API call [404]' in e.args[0]:
-            raise DemistoException("Invalid ID for one or more fields that request IDs \n"
-                                   "Please make sure all IDs are correct")
-        else:
-            raise DemistoException(e.args[0])
+    response = client.get_custom_fields_request()
+    custom_fields_response = response['custom_fields']
+    headers = ['id', 'name', 'customized_type', 'field_format', 'regexp', 'max_length', 'is_required', 'is_filter',
+               'searchable', 'trackers', 'issue_categories', 'enabled_modules', 'time_entry_activities',
+               'issue_custom_fields']
+    # Some custom fields are numbers and tableToMarkdown can't transform it
+    for custom_field in custom_fields_response:
+        custom_field['id'] = str(custom_field['id'])
+        custom_field['is_required'] = str(custom_field['is_required'])
+        custom_field['is_filter'] = str(custom_field['is_filter'])
```

Comment:
It is because the `is_auto_json_transform=True`, lets open a dev-bug ticket to fix this 😊